### DISCUSSION TITLE
Allow to use '{ }' to enclose blocks of code

### DISCUSSION THREAD

**Discussion by azur-wolve** [godotengine/godot-proposals#4760] [Scripting]
Instead of necessarily use `:` for scopes of code blocks
allow to optionally use bodies enclosed by curly brackets `{ ... }`

So both sides (Pythonic and C) would be happy.

**Comment by Calinou** [comment]
This was already proposed multiple times in the past, and it was rejected: https://github.com/godotengine/godot-proposals/issues/1555

**Comment by dalexeev** [comment]
We prefer not to have multiple syntax options _doing the same thing_. For example, we have two dictionary styles, but the Lua style allows you to omit quotes for keys. We have two setter/getter styles, but the alternative style allows you to use a separate function like `setget` in 3.x. An alternative function declaration syntax would not bring any bonuses other than aesthetic ones. I think it's not worth it, it would only complicate parser, clutter documentation, etc.

The alternative syntax _could_ be seen as providing additional capabilities, such as being more convenient for code generation. However, this is quite controversial and not well justified.

Also, judging by the reactions and activity over almost 2 years, this proposal did not receive support. So I think it's reasonable to close this discussion. Thanks for the contribution nonetheless!
